### PR TITLE
KOGITO-4081 - restore dashboards docker compose IT test

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/KOGITO-4081

This PR aims to restore the docker compose test for the dashboards. 

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: dmn-drools-quarkus-metrics/pom.xml [hunk 1]
```diff
@@ -103,6 +110,25 @@
...
             </configuration>
           </execution>
+          <execution>
+            <id>copy-docker-compose-for-tests</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.testOutputDirectory}</outputDirectory>
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/pom.xml:120]: Can you please make sure that dashboards are copied/used from `META-INF/resources/monitoring/dashboards/` folder and not from `dashboard` folder one?
This second folder was used only in this test and not by the operator so I removed it with [this PR](https://github.com/kiegroup/kogito-runtimes/pull/958/files#diff-4e55e3fdbe846134a9dee541dd6f96729938655b38f34f712f2122738a109b7d)

**Comment by r00ta** [comment]
Jenkins retest this please

File: dmn-drools-quarkus-metrics/pom.xml (hunk 1 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/pom.xml:120]: Atm this PR is green with master, but with your changes this new test is going to fail. We can move forward with this PR and then I can create another one that is related to your changes

File: dmn-drools-quarkus-metrics/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,2 +1,5 @@
-# Configuration file
-# key = value
\ No newline at end of file
+# Container image
+quarkus.container-image.build=true
+quarkus.container-image.group=org.kie.kogito
+quarkus.container-image.name=dmn-drools-quarkus-metrics-example
+quarkus.container-image.tag=2.0.0-SNAPSHOT
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/src/main/resources/application.properties]: Can you use a "stable" tag? It is internal just for integration testing so you can use something like `1.0-INTEGRATIONTEST` or similar

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java [hunk 3]
```diff
@@ -16,31 +16,49 @@
...
-    public static DockerComposeContainer environment =
-            new DockerComposeContainer(new File("./docker-compose.yml"))
-                    .withExposedService("grafana_1", 3000, Wait.forHttp("/")
-                            .forStatusCode(200));
+    public static DockerComposeContainer environment;
+
+    static {
+        try {
+            environment = new DockerComposeContainer(new File(GrafanaDockerComposeIT.class.getClassLoader().getResource("./docker-compose.yml").toURI()))
+                    .withExposedService("grafana_1", 3000, Wait.forListeningPort().withStartupTimeout(Duration.ofMinutes(8)));
```
Comment (jiripetrlik) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java:52]: Just for curiosity. How did you estimate 8 minutes timeout?

**Review by jiripetrlik — COMMENTED** [review]
Looks good. I was able to run GrafanaDockerComposeIT successfully for 3 time in row. However, I'm not sure whether this PR is assigned to the correct JIRA. Because JIRA is about "dmn-drools-quarkus-example" but this PR fixes integration test in dmn-drools-quarkus-metrics example.

**Comment by danielezonca** [comment]
> dmn-drools-quarkus-metrics

@jiripetrlik 
Good catch! Updated Jira title

**Review by jiripetrlik — APPROVED** [review]
Thank you for updating JIRA title.

**Review by danielezonca — APPROVED** [review]

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java (hunk 3 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java:52]: Hi @jiripetrlik , the grafana container is a bit slow to start up and the default timeout is exceeded. I did some local test and it takes ~1minutes. I thought x8 was a safe option considering that the build agent might be slower.

Comment (jiripetrlik) [reply dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java:52]: Understand, thank you for info.

**Pull request merged**